spec/take_away_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'take_away'
+
+describe TakeAway do
+  context '#read_menu' do
+    it 'exists' do
+      expect(subject).to respond_to(:read_menu)
+    end
+
+    it 'returns an array of menu items' do
+      expect(subject.read_menu).to be_instance_of(Array)
+    end
+  end
+
+  context '#order' do
+    it 'selects dish' do
+      allow(subject).to receive(:menu).and_return([{ :index => 1, :item => "Miso Soup", :amount => "2.00" }])
+      item_number = 1
+      expect(subject.order(item_number)).to eq("You have added 1 x Miso Soup to your basket.")
+    end
+
```

Comment:
Do you need to stub here? Can you not just load your @menu, and then call menu on your subject? You are not using another class in this test right?